src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -595,7 +595,23 @@ extension SecureStore: CoronaTestStoringLegacy {
 		get { kvStore["isSubmissionConsentGiven"] as Bool? ?? false }
 		set { kvStore["isSubmissionConsentGiven"] = newValue }
 	}
+}
 
+extension SecureStore: AllowListCaching {
+	var ticketValidationAllowList: TicketValidationAllowList {
+		get {
+			kvStore["ticketValidationAllowList"] as TicketValidationAllowList? ?? TicketValidationAllowList(
+				validationServiceAllowList: [],
+				serviceProviderAllowList: []
+			)
+		}
+		set { kvStore["ticketValidationAllowList"] = newValue }
+	}
+	
+	var serviceProviderAllowlist: [Data] {
+		get { kvStore["serviceProviderAllowlist"] as [Data]? ?? [] }
+		set { kvStore["serviceProviderAllowlist"] = newValue }
+	}
```

Comment:
won't this double the cache? It's already stored inside the cached rest service.

Reply:
I wasnt sure about this, i thought we only store the eTag, how can can i access the cached stuff ?